user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Maps.md

```diff
@@ -0,0 +1,70 @@
+---
+uid: DashboardMaps
+---
+
+# Maps
+
+This component is used to display markers and/or lines on a map. It uses a GQI query as data input. 
+
+> [!NOTE]
+> To use the Maps component, the host servers for DataMiner Maps have to be configured in the file *C:\Skyline DataMiner\Maps\ServerConfig.xml*. If this file does not exist, it will be created automatically when using a Maps component for the first time and it will contain the default configuration. To change the configuration, see [Configuring the DataMiner Maps host servers](xref:Configuring_the_DataMiner_Maps_host_servers)
```

Comment:
Would not mention the default configuration (is implied by having it automatically created).

Reply:
Do we create that file automatically?

Reply:
> Do we create that file automatically?

According to the docs in the reference we do